andela-aabdulwahab/pilxel
# Move the Develop branch into Master

#### What does this PR do?

Move the develop branch to master
#### Description of Task to be completed?

After complete development of the application, the develop branch is moved to master to prepare for the first release
#### How should this be manually tested?

Visit the [Application](https://pilxel.herokuapp.com/) the see it's current state
#### Any background context you want to provide?

This mark the completion of checkpoint Four, awaiting trainers review
